cfme/tests/cloud_infra_common/test_policy_simulation.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+"""This module tests events that are invoked by Cloud/Infra VMs."""
+import pytest
+
+from cfme.cloud.provider import CloudProvider
+from cfme.common.vm_views import PolicySimulationDetailsView
+from cfme.common.vm_views import PolicySimulationView
+from cfme.infrastructure.provider import InfraProvider
+from cfme.markers.env_markers.provider import ONE_PER_CATEGORY
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.blockers import BZ
+
+pytestmark = [
+    pytest.mark.usefixtures('uses_infra_providers', 'uses_cloud_providers', 'setup_provider'),
+    pytest.mark.tier(2),
+    pytest.mark.provider(
+        classes=[InfraProvider, CloudProvider],
+        required_fields=[(['cap_and_util', 'chargeback_vm'], 'cu-24x7')],
+        selector=ONE_PER_CATEGORY,
+        scope="module"
+    ),
+]
+
+
+@pytest.mark.parametrize("navigation", ["provider_vms", "vm_summary"])
+def test_policy_simulation_ui(appliance, provider, navigation):
+    """
+    Bugzillas:
+        * 1670456, 1686617, 1686619, 1688359, 1550503
+    Polarion:
+        assignee: jdupuy
+        casecomponent: Control
+        caseimportance: medium
+        initialEstimate: 1/6hr
+    """
+    collection = provider.appliance.provider_based_collection(provider)
+    vm = collection.instantiate('cu-24x7', provider)
+    if navigation == "provider_vms":
+        if BZ(1686617).blocks or BZ(1686619).blocks:
+            pytest.skip("Skipping due to blocker BZs, {}, {}".format(BZ(1686617), BZ(1686619)))
+        view = navigate_to(vm.provider, "ProviderVms")
+        # Click the checkbox for our VM
+        view.entities.get_entity(name=vm.name, surf_pages=True).check()
+        # Navigate to policy simulation page
+        view.toolbar.policy.item_select("Policy Simulation")
+        view = vm.create_view(PolicySimulationView, wait="10s")
```

Comment:
Optional:
I have one thought if you like please have a glance.
I would write one more navigation against `vm` `AllForProviderSimulationView`.

Reply:
@digitronik, I feel that this navigation is atypical. Since this test is specifically in reference to a BZ, I think it is good to be explicit about the navigation steps here too.

Reply:
I do think that there should be another view that is registered against a vm collection, so that we can select multiple vms for a policy simulation. However, I'd like to implement that in a different PR.